### DISCUSSION TITLE
Linux (Manjaro) - Device tab black / blank

### DISCUSSION THREAD

**Discussion by TexZeTech** [OrcaSlicer/OrcaSlicer#638] [Q&A] [answered]
So I have been using OrcaSlicer and today I went to use it after a system update in Manjaro and my device tab just results in a blank window as shown below. It works fine in my ubuntu VM. Anyone else have this issue or have an idea of how to fix it?

![Screenshot_20230404_113305](https://user-images.githubusercontent.com/7254540/229872111-56d0d43a-0add-4bff-a5cf-3eac36469ee7.png)

I have removed the appimage removed my config, reintegrated the app image, and even used the [AUR package ](https://aur.archlinux.org/packages/orca-slicer) (I will probably stay with its aur package)

**System info:**
Operating System: Manjaro Linux 
KDE Plasma Version: 5.27.3
KDE Frameworks Version: 5.104.0
Qt Version: 5.15.8
Kernel Version: 6.1.21-1-MANJARO (64-bit)
Graphics Platform: X11
Processors: 8 × Intel® Core™ i7-3770K CPU @ 3.50GHz
Memory: 15.6 GiB of RAM
Graphics Processor: NVIDIA GeForce GTX 1080/PCIe/SSE2

**Comment by TexZeTech** [comment]
well, I cannot replicate it on my fresh manjaro install in a VM.... great....

**Reply by TexZeTech** [reply]
So I can replicate this .... it worked on the live disk but not the fully installed one.... hmmmm

**Comment by TexZeTech** [comment]
Just integrated the appimage from the V1.6.2 stable Ubuntu build & the device tab is still blank on my Manjaro system. I'm not sure what is required to use that page.

@SoftFever do you have any idea what packages are getting called into use by that window? Because if there is no IP address the video showing how to use it plays and it was working until a system update so clearly something that was in use isn't in my system anymore any ideas?

**Reply by SoftFever** [reply]
hmmm, it's using libwebkit.
But I have no idea why this happens

**Reply by TexZeTech** [reply]
This makes two of us but now I have a direction to go down.

**Comment by TexZeTech** [comment]
I've rased an [issue here](https://github.com/SoftFever/OrcaSlicer/issues/920) as I don't know how to proceed and would like this to work.

**Comment by Anathema-Device** [comment]
I am seeing the blank home, Device and Project pages as well.  It was working, went for supper, came back to the PC and  and the application seemed unresponsive.  Closed it with the X and now Orcaslicer works only on the Prepare and Preview pages and the rest show blank.  Restarting Orcaslicer makes no difference.  This is on fully updated Windows 11, 3060 graphics card and Orcaslicer 1.6.3

**Comment by misscelestia** [comment]
I am also having this issue on my Mac running Sonoma. The device tab is just blank, I can send jobs to my printer. It is connected via OctoPrint and Orca can confirm successful connectivity to the IP via application key every time I test it.

**Comment by pablunsky** [comment]
I solved this issue by reconfiguring the printer connection.

In Printer config, click on the "wifi" green icon, entered connection parameters and the tab was fixed.

<img width="436" alt="image" src="https://github.com/SoftFever/OrcaSlicer/assets/26340100/8ec90e89-3fdb-49e4-ad06-b7081d09973f">

**Comment by TexZeTech** [comment, marked as answer]
https://github.com/SoftFever/OrcaSlicer/issues/920#issuecomment-2025895738

solved for me.

**Comment by donalffons** [comment]
I'm also having this issue - the suggestion in https://github.com/SoftFever/OrcaSlicer/issues/920#issuecomment-2025895738 doesn't work for me. Meanwhile, I'm running the application via `WEBKIT_DISABLE_COMPOSITING_MODE=1 nohup orca-slicer &`, which seems to be a decent workaround for me.
